app/javascript/components/ConversationPage.js

```diff
@@ -7,24 +7,19 @@ import Message from './Message';
 import { postData } from './utils/sendData';
 import FormData from './utils/FormData';
 import METHODS from './utils/RestConstants';
+import Header from './reusable/Header';
```

Comment:
These need to be removed now that the Header and Footer are in the WrapperComponent

Reply:
Will do.